.github/workflows/docbuild.yml

```diff
@@ -23,10 +23,10 @@ jobs:
         python -m pip install --upgrade pip
         pip install -r requirements.txt
         pip install -r requirements-dev.txt
-        pip install .
+        pip install -r requirements-fury.txt
     - name: Install software
       run: |
-        pip install .
+        pip install -e .
```

Comment:
For some reason, the standard install was running into errors. It could not find the new folder I added. However I will try this change and see if it runs into the same error.

Reply:
Did you add  an `__init__.py` file in that folder? It has to have that to be considered a Python module.

Reply:
I think that may be the problem! I will do that.

Reply:
@arokem  Now I am experiencing a new issue.
`pip install '.[dev,fury]'` causes the following warnings:
```
WARNING: pyAFQ 0.1.dev119761714318 does not provide the extra 'dev'
WARNING: pyAFQ 0.1.dev119761714318 does not provide the extra 'fury'
```
However, pip install -e '.[dev,fury]' and pip install 'pyAFQ[dev,fury]' do not cause these warnings. 

These warnings are what is causing the PR to fail the test suite (its not installing dev or fury).

It is exactly the same as what they describe here, but this is 5 years old and marked resolved (my pip is updated):
https://github.com/pypa/pip/issues/1236

Reply:
Sorry - more questions here. I see that you wrote `pip install '. [dev, fury]'`. 

Do you really have those quotes in there? What about `pip install .[dev,  fury]` (no quotes)?

Reply:
it returns bad pattern on zsh and invalid requirement on bash. If you do no quotes and no spaces, It works on bash but not zsh. With quotes works on both. I have not tested other shells.